step.yml

```diff
@@ -61,19 +61,24 @@ inputs:
       title: "Send To emails"
       description: |
         Comma separated email address list.
-      summary: ""
+      summary: "
```

Comment:
for empty string please use `""` - or just remove the `summary` item, it's an empty string by default ;)